### DISCUSSION TITLE
Selecting "Not interested in <tag>" keeps the article visible

### DISCUSSION THREAD

**Discussion by dcraig327** [dailydotdev/daily#905] [Feature Requests]
On My Feed. If I select "Not interested in #social-media" in the image below. 

![chrome_FG3bKFUKZe](https://github.com/dailydotdev/daily/assets/37514553/4e8fd5eb-4eae-4bb9-ab46-500ca581a14a)

The entire article will close as if I hit the X icon. It's unlikely that article is still visible in the cards loaded in My Feed. Yet for functionality, I may also want to select not interested in the other tags/publisher.

This would reduce the need for a downvote button (I wasn't able to scan all 11 pages of requests):
https://github.com/dailydotdev/daily/discussions/904
https://github.com/dailydotdev/daily/discussions/460

Why? There's two ways to setup "My Feed". I can select exactly what I'm interested in, I can select everything and remove what I'm not interested in. The former creates a type of exploration bias and can't see beyond what you like to know what you may like. The latter requires an investment of time to customize what you would like to see and I'd even say if there's a topic I'm interested in the future, but not now, I'd remember that and add that tag later.
EDIT: The act of removing a tag is teaching me the tag exists and provides a visual example of an article referencing the tag.

**Comment by idoshamun** [comment]
@dcraig327 thanks for sharing your perspective. It's an interesting one that I haven't thought about.
Food for thought, no doubt :)
And regarding the issue, we will fix it
